app/views/application_letters/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <%- model_class = ApplicationLetter -%>
 <div class="page-header">
-  <h1><%=t '.title', :default => model_class.model_name.human.pluralize.titleize %></h1>
+  <h1><%=t '.title', :default => model_class.model_name.human(:count => model_class.count).titleize %></h1>
```

Comment:
It is what I intended at least.

I talked with our PO @tobiduer today and he said that he does not care about this detail, so this shpuld be fine.
Apparently, the POs are talking about this page anyway, since students should only see their own application letters and all others only need to see them as part of an event. But this is not part of any user story right now, it is currently discussed. Ergo this is not part of our acceptamce criteria.